File: services/python/tests/ingest/fixtures/monorepo/web/src/service.ts
import type {
  Client,
} from "@example/client";
import "./polyfill";
export { helper } from "./helper";

export interface Repository {
  find(id: string): Promise<string>;
}
export class Service {
  async find(
    id: string,
  ): Promise<string> {
    return id;
  }
}
export const load = async (
  id: string,
): Promise<string> => id;
// function fakeMethod() {}

File: services/python/tests/ingest/fixtures/monorepo/worker/service.go
package worker

import (
	"context"
	db "example.com/storage/client"
)

type Service struct{ Client *db.Client }

func (s *Service) Find(
	ctx context.Context,
	query string,
) (string, error) {
	return query, nil
}

func New() *Service { return &Service{} }
